src/app/clients/getExpandedClientDetails.ts

```diff
@@ -0,0 +1,199 @@
+            delivery_instructions,
+            address_1,
+            address_2,
+            address_town,
+            address_county,
+            address_postcode,
+
+            family:families(
+                age,
+                gender
+            ),
+
+            dietary_requirements,
+            feminine_products,
+            baby_food,
+            pet_food,
+            other_items,
+            extra_information
+        )
+
+    `
+        )
+        .eq("primary_key", parcelId)
+        .single();
+
+    return response.data;
+};
+
+export interface ExpandedClientDetails extends Data {
+    "voucher_#": Schema["parcels"]["voucher_number"];
+    full_name: Schema["clients"]["full_name"];
+    phone_number: Schema["clients"]["phone_number"];
+    packing_date: string;
+    packing_time: string;
+    delivery_instructions: Schema["clients"]["delivery_instructions"];
+    address: string;
+    household: string;
+    "age_&_gender_of_children": string;
+    dietary_requirements: string;
+    feminine_products: string;
+    baby_products: Schema["clients"]["baby_food"];
+    pet_food: string;
+    other_requirements: string;
+    extra_information: Schema["clients"]["extra_information"];
+}
+
+export const rawDataToExpandedClientDetails = (
+    rawClientDetails: RawClientDetails
+): ExpandedClientDetails => {
+    if (rawClientDetails === null) {
+        return {
+            "voucher_#": "",
+            full_name: "",
+            phone_number: "",
+            packing_date: "",
+            packing_time: "",
+            delivery_instructions: "",
+            address: "",
+            household: "",
+            "age_&_gender_of_children": "",
+            dietary_requirements: "",
+            feminine_products: "",
+            baby_products: null,
+            pet_food: "",
+            other_requirements: "",
+            extra_information: "",
+        };
+    }
+
+    const client = rawClientDetails.client!;
+
+    return {
+        "voucher_#": rawClientDetails.voucher_number,
+        full_name: client.full_name,
+        phone_number: client.phone_number,
+        packing_date: formatDatetimeAsDate(rawClientDetails.packing_datetime),
+        packing_time: formatDatetimeAsTime(rawClientDetails.packing_datetime),
+        delivery_instructions: client.delivery_instructions,
+        address: formatAddressFromClientDetails(client),
+        household: formatHouseholdFromFamilyDetails(client.family),
+        "age_&_gender_of_children": formatBreakdownOfChildrenFromFamilyDetails(client.family),
+        dietary_requirements: client.dietary_requirements.join(", "),
+        feminine_products: client.feminine_products.join(", "),
+        baby_products: client.baby_food,
+        pet_food: client.pet_food.join(", "),
+        other_requirements: client.other_items.join(", "),
+        extra_information: client.extra_information,
+    };
+};
+
+export const formatDatetimeAsTime = (datetime: string | null): string => {
```

Comment:
same comments as from getClientsTableData

Reply:
I've changed this to use `.toLocaleTimeString()`